health_check.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
```

Comment:
Let's drop this file. We were fetching more stats (e.g., coverage), but it seems we do not run it as part of CI. Easier to just drop it for now.

Reply:
OK